TU-Agent/tygron.act2g

```diff
@@ -38,17 +38,18 @@ define building_plan_upgrade(UpgradeID, MultiPolygon) with
 	
 % Define the answer to a popup action
 % Answer another stakeholders request
-% params = RequestID, AnswerID
-define popup_answer(RequestID, AnswerID) with
-  pre{ requests(X), not(empty(X)) }
-  post{ requestAnswered(RequestID, AnswerID) } 
+% params = PopupID, AnswerID
+define popup_answer(PopupID, AnswerID) with
+  pre{not(requestAnswered(_, PopupID))}
+  post{ requestAnswered(PopupID, AnswerID) } 
```

Comment:
Looks like it.
Agent did fine in the simulation and it doesn't throw any errors about the "not(..)"